test/server_test.py

```diff
@@ -39,11 +39,11 @@
 port = 0
 
 [local]
-socket =
+socket = {run_dir}/local.sock
```

Comment:
The local socket is never unix socket, it is always an absract unix socket - using file path is incorrect since this is not the way the socket is used.

The user or the tests never need to know the path of the socket, we report the socket in OPTIONS request.